borgmatic/borg.py

```diff
@@ -28,10 +30,13 @@ def create_archive(
 ):
     '''
     Given an excludes filename (or None), a vebosity flag, a storage config dict, a space-separated
-    list of source directories, a local or remote repository path, and a command to run, create an
-    attic archive.
+    list of source directories (spaces escaped with a backslash not treated as separators), a local
+    or remote repository path, and a command to run, create an attic archive.
     '''
-    sources = re.split('\s+', source_directories)
+    # To support escaping spaces in directory name, temporarily replace them
+    temp_source_directories = source_directories.replace(ESCAPED_SPACE, TEMP_REPLACEMENT_FOR_ESCAPED_SPACE)
+    sources = re.split('\s+', temp_source_directories)
+    sources = [s.replace(TEMP_REPLACEMENT_FOR_ESCAPED_SPACE, ' ') for s in sources]
```

Comment:
You could accomplish the same thing perhaps a little more succinctly with a negative look-behind regular expression. Here's an example:

```
sources = [source.replace('\\ ', ' ') for source in re.split(r'(?<!\\) ', source_directories)]
```

That `"r'(?<!\\) '` matches any non-backslash character followed by a space, but without capturing that preceding character.

More here: https://docs.python.org/3/library/re.html